internal/runtime/hcsv2/nvidia_utils.go

```diff
@@ -23,22 +24,32 @@ const lcowNvidiaMountPath = "/run/nvidia"
 const annotationContainerGPUCapabilities = "io.microsoft.container.gpu.capabilities"
 const nvidiaDebugFilePath = "/nvidia-container.log"
 
+const nvidiaToolBinary = "nvidia-container-cli"
+
 // TODO katiewasnothere: prestart hooks will be depracated, this needs to be moved to a createRuntime hook
 // described here: https://github.com/opencontainers/runtime-spec/blob/39c287c415bf86fb5b7506528d471db5405f8ca8/config.md#posix-platform-hooks
 // addNvidiaDevicePreHook builds the arguments for nvidia-container-cli and creates the prestart hook
 func addNvidiaDevicePreHook(ctx context.Context, spec *oci.Spec) error {
-	nvidiaToolBinary := "nvidiaPrestartHook"
-	nvidiaToolPath, err := exec.LookPath(nvidiaToolBinary)
+	genericHookBinary := "genericHook"

```

Comment:
is this supposed to be lower case as well?

Reply:
Apparently that file didn't save when I edited it lol fixed